.ai/flowable-schema-research-prompt.md

```
@@ -0,0 +1,410 @@
+    open fun customProcessEngineConfiguration(
+        dataSource: DataSource,
+        transactionManager: PlatformTransactionManager
+    ): SpringProcessEngineConfiguration {
+        return SpringProcessEngineConfiguration().apply {
+            // COMPLETE working configuration here
+        }
+    }
+}
+```
+
+**OR**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)